DeviceDetector.php

```diff
@@ -916,9 +916,9 @@ protected function parseDevice(): void
         $clientName = $this->getClientAttribute('name');
 
         /**
-         * if it's fake UA then it's best not to identify it as Apple running Android OS
+         * if it's fake UA then it's best not to identify it as Apple running Android OS or GNU/Linux
          */
-        if ('Android' === $osName && 'Apple' === $this->brand) {
+        if ('Apple' === $this->brand && \in_array($osName, ['Android', 'GNU/Linux'])) {
```

Comment:
```suggestion
        if ('Apple' === $this->brand && \in_array($osName, ['Android', 'GNU/Linux', 'Windows'])) {
```

I don't know if is much better to check in the array of Apple operating systems?
```suggestion
        if ('Apple' === $this->brand && !\in_array($osName, ['iPadOS', 'tvOS', 'watchOS', 'iOS', 'Mac'])) {
```
For example I have one with `Windows`.

```YML
-
  user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 13_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/79.0.3945.73 Mobile/15E148 Safari/604.1
  headers:
    brands:
      - brand: Not_A Brand
        version: '99'
      - brand: Chromium
        version: '92'
    mobile: '0'
    platform: Windows
    platformVersion: "10.0"
    uaFullVersion: 92.0.4515.131
    architecture: 'x86'
  os:
    name: Windows
    version: "10"
    platform: x86
  client:
    type: browser
    name: Chrome Mobile iOS
    version: 79.0.3945.73
    engine: WebKit
    engine_version: 605.1.15
  device:
    type: desktop
    brand: ""
    model: ""
  os_family: Windows
  browser_family: Chrome
```